### PR TITLE
[FTAMS102020] Sophie Tichelaar

### PR DESCRIPTION
**Pull request by Sophietic**
til 4.2

### CODE REVIEW

**Comment by Sophietic** [comment]
Mandatory iterations done

File: src/functions-and-arrays.js [hunk 1]
```diff
@@ -1,19 +1,69 @@
...
+findLongestWord(words)
+
+// Iteration #3: Calculate the sum 
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+function sumNumbers(array){
+  let sum =0;
+  for (let i=0; i<array.length; i++){
+    sum = sum + array[i]
```
Comment (LloydC) [inline src/functions-and-arrays.js:35]: Nice! Another way to write it would be `sum += array[i]`

File: src/functions-and-arrays.js [hunk 2]
```diff
@@ -47,6 +134,17 @@ const wordsCount = [
...
+  let count =0
+  for(let i=0; i<array.length; i++){
+if (array[i]===word){
+  count++
+}
+
+  }
+  return count
+}
+
```
Comment (LloydC) [inline src/functions-and-arrays.js:147]: Nicely done Sophie!

An alternative solution is 
`function howManyTimes(arr, word) {

  let count = 0;
  for (let el of arr) {
    if (el === word) count++;
  }
  return count;
}`

File: src/functions-and-arrays.js [hunk 3]
```diff
@@ -29,9 +79,46 @@ const wordsUnique = [
   'bring'
 ];
```
Comment (LloydC) [inline src/functions-and-arrays.js:81]: An alternative solution is `
const uniquifyArray = words => {
  let uniqueArr = [];
  for (let word of words) {
    if (!uniqueArr.includes(word)) uniqueArr.push(word);
  }
  return uniqueArr;
};`

**Pull request closed without merging**